package.json

```diff
@@ -79,7 +76,6 @@
 		"js-cookie": "^2.2.1",
 		"jsonwebtoken": "^8.5.1",
 		"lodash": "^4.17.15",
-		"lodash-es": "^4.17.15",
```

Comment:
I compared using 'lodash-es' imports vs 'lodash' and it actually increased the bundle size

Reply:
😲

Reply:
Curious. I had assumed using es modules would give us tree-shaking and therefore less lodash in the final bundle, but I guess not